ngirimana/MyDiary-Adc
# #169519111 add signup API by storing data in database

> #### What does this PR do?
* add signup endpoint
> #### Description of Task to be completed
- add signup endpoint
- add test for endpoint
- add validation for user data
> #### How should this be manually tested?
* Clone the repo
* run ``` cd MyDiaru-Adc``` in terminal
* checkout  **ft-signup-db-api-169519111**  brabch by running ```git checkout ft-signup-db-api-169519111``` in terminal
* use 'npm install' to install all necessary dependencies
* set the environmental variable in```.env``` file
* Run 'npm run dev-start' to run the server 
* use Postman and send  post request to ```localhost:4000/api/v2/auth/signup```
> #### Any background context you want to provide?
*  you should install nodejs and postgres before running the reopsitory
> #### What are the relevant pivotal tracker stories?
* [ #169519111](https://www.pivotaltracker.com/story/show/169519111)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38111965/68075738-7297c380-fdb4-11e9-9b6e-23dbb8faff73.png)
![image](https://user-images.githubusercontent.com/38111965/68075767-06698f80-fdb5-11e9-9d1a-bdca5147b2dc.png)
